app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -103,12 +124,8 @@ export class MergeConflictsDialog extends React.Component<
    *  dismisses the modal and shows the abort merge warning modal
    */
   private onCancel = async () => {
-    const anyResolvedFiles = getUnmergedFiles(this.props.workingDirectory).some(
-      f =>
-        isConflictedFile(f.status) &&
-        f.status.lookForConflictMarkers &&
-        f.status.conflictMarkerCount === 0
-    )
+    const anyResolvedFiles =
+      getResolvedFiles(this.props.workingDirectory).length > 0
```

Comment:
Tiny nitpick, seeing as we're only using this to check if any files are resolved we can rename the `getResolvedFiles` method to `anyResolvedFiles` and change the `.filter` to a `.some` rather than iterating through all of them. That said this is tremendously unimportant for this PR but if we're iterating here in the future I think that would be a good change.